### PR TITLE
doc-700

### PR DESCRIPTION
**Pull request by resmaa**
https://spectrocloud.atlassian.net/browse/DOC-700

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1092#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 02aa4112df4d96f2e346cd5e6df88aea50be2a85 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1092/librarium_build/1617833430114897920) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md
Comment (ritawatson) [inline content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md:726]:
```suggestion
|Virtual Cluster Admin |Provides administrative privilege to perform all virtual cluster operations on App resources.|
```

File: content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md
Comment (ritawatson) [inline content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md:727]:
```suggestion
|Virtual Cluster Editor|Allows the user to perform edit operations on a virtual cluster but not to create or delete a virtual cluster.|
```

File: content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md
Comment (ritawatson) [inline content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md:728]:
```suggestion
|Virtual Cluster Viewer|Allows the user to view all the virtual cluster resources but not to modify them.|
```

**Review by ritawatson — APPROVED** [review]
Descriptions of roles are repetitive because role names is repeated in there. I offered suggestions to remove the repetition. 

It would be good to apply this in other docs not included in this PR that have the same type of repetition in  role descriptions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1092#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1092#pullrequestreview-1268457190" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
